config.yaml.example

```diff
@@ -1,83 +1,37 @@
-  perfect_aeon_kills    : False  # Before YuYevon, True is slower but more swag.
-  attempt_djose         : False  # Try Djose skip? (not likely to succeed)
-  legacy_soundtrack     : True   # use the original Soundtrack instead of arranged
-  do_not_skip_cutscenes : False  #            - Missing documentation -
-  battle_speedup        : False  # Use 4x speed when doing battles.
-
-  # Accessibility for blind
-  skip_cutscene_flag    : True   #            - Missing documentation -
-  skip_diag_flag        : True   # Set to same value as skip_cutscene_flag
-  play_TTS_flag         : False  #            - Missing documentation -
-  rails_trials          : True   #            - Missing documentation -
-  rails_egg_hunt        : True   #            - Missing documentation -
-
-  # Blitzball
-  force_blitz_win       : False  # Sets Aurochs score to 50 at halftime, testing only.
-  blitz_loop            : False  # Loop on the same seed immediately after Blitzball.
-  blitz_loss_force_reset: True   # True == reset after blitz loss
-  blitz_win_value       : True   # No default value required
-  blitz_overtime        : False  # Set to False, no need to change ever.
-  blitz_first_shot_val  : False  #            - Missing documentation -
-  oblitz_attack_val     : "255"  # Used for RNG manip tracking
-
-  # Sphere grid
-  full_kilik_menu       : False  # Default to False
-  early_tidus_grid_val  : False  # Default False
-  early_haste_val       : -1      # Default -1
-  wakka_late_menu_val   : False  # Default False
-  end_game_version_val  : 0      # Default 0
-
-  # Equipment
-  zombie_weapon_val     : 255    # Default 255
-  l_strike_count        : 0      # Default 0
-
-  # RNG Manip
-  yellows               : 0      #            - Missing documentation -
-  confirmed_seed_num    : 999    #            - Missing documentation -
-  skip_zan_luck         : False  #            - Missing documentation -
-
-  # Other
-  new_game              : False  #            - Missing documentation -
-  self_destruct         : False  # Default False
-  ytk_farm              : 0      # Default to 0
-  rescue_count          : 0      # Default to 0
-  flux_overkill_var     : False  # Default to False
-  try_ne_val            : True   # Based on
-  ne_armor_val          : 255    # Default 255
-  ne_battles            : 0      # Default to 0
-  nea_zone              : 0      # Decides in which zone we charge Rikku OD after reaching Zanarkand.
-  generate_saves        : False  # Generates a full set of saves throughout the run.
-  # first_hits is still in vars.py
-  
-  # Nemesis variables, unused in any%
-  nem_ap_val            : 1      # Nemesis route, determines Tidus level-up progress. Default to 1
-  yojimbo_index         : 1      # Used in arena battles to track Zanmatou progress.
-  # area_results is still in vars.py
-  # species_results is still in vars.py
-  # original_results is still in vars.py
+  # Game modes:
```

Comment:
This file ought to include all possible configuration options, and they should have defaults which lead to an any% run. It looks like a bunch of values were deleted

Reply:
The VARS file contains all possible config options. The goal is to make game mode so that it's user-friendly.